libraries/ESP8266HTTPClient/examples/StreamHttpsClient/StreamHttpsClient.ino

```diff
@@ -0,0 +1,116 @@
+/**
+   StreamHTTPClient.ino
+
+    Created on: 24.05.2015
+
+*/
+
+#include <Arduino.h>
+
+#include <ESP8266WiFi.h>
+#include <ESP8266WiFiMulti.h>
+
+#include <ESP8266HTTPClient.h>
+
+#define USE_SERIAL Serial
+
+ESP8266WiFiMulti WiFiMulti;
+
+void setup() {
+
+  USE_SERIAL.begin(115200);
+  // USE_SERIAL.setDebugOutput(true);
+
+  USE_SERIAL.println();
+  USE_SERIAL.println();
+  USE_SERIAL.println();
+
+  for (uint8_t t = 4; t > 0; t--) {
+    USE_SERIAL.printf("[SETUP] WAIT %d...\n", t);
+    USE_SERIAL.flush();
+    delay(1000);
+  }
+
+  WiFi.mode(WIFI_STA);
+  WiFiMulti.addAP("SSID", "PASSWORD");
+
+}
+
+void loop() {
+  // wait for WiFi connection
+  if ((WiFiMulti.run() == WL_CONNECTED)) {
+
+    HTTPClient http;
+
+    BearSSL::WiFiClientSecure client;
+
+    bool mfln = client.probeMaxFragmentLength("tls.mbed.org", 443, 1024);
+    USE_SERIAL.printf("\nConnecting to https://tls.mbed.org\n");
+    USE_SERIAL.printf("MFLN supported: %s\n", mfln ? "yes" : "no");
```

Comment:
MFLN is not a commonly used acronym, suggest spelling out "Maximum fragment length negotiation".

Reply:
Replacing MFLN with "Maximum fragment length negotiation"